src/content/en/fundamentals/primers/service-workers/a-basic-service-worker.markdown

```diff
@@ -0,0 +1,170 @@
+---
+layout: shared/narrow
+published: false
+title: "Setting up a basic service worker"
+description: "This section describes the most basic service worker possible. It shows you how a client gets, or registers, a service worker. It shows you how a service worker prepares to act as a proxy for its clients. Both subjects have more depth than is shown in this section. It's not intended to be comprehensive; it's intended to give you a basic foundation on which to build service worker knowledge."
+authors:
+  - josephmedley
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 4
+key-takeaways:
+  tldr:   
+  - "Call <code>register()</code> to get a service worker." 
+  - "Learn to use promises before attempting service workers."
+  - "Service workers have a limited scope and require HTTPS."
+  - "The installation process has two events: install and activate."
+  - "To start proxying, a service worker needs a call to <code>clients.claim()</code> or a user navigation."
```

Comment:
"or a user navigation." → "or the user needs to navigate to a new page in scope."

Or some other phrasing, because it sounds incomplete as currently phrased.